telegram/ext/updater.py

```diff
@@ -251,11 +255,54 @@ def start_polling(self,
         Returns:
             :obj:`Queue`: The update queue that can be filled from the main thread.
 
+        Raises:
+            ValueError: If :attr:`clean` is set as :obj:`datetime.timedelta` and is < 1 second.
+
         """
         with self.__lock:
             if not self.running:
                 self.running = True
 
+                self.logger.debug('clean:               "%s"', type(clean))
```

Comment:
Logging should suffice in the `bootstrap_*` methods

Reply:
part of cleanup once y'all are happy with the code itself. I'll wrap it in a comment to make it clearer this is just for debugging while developing